packages/common/src/types/openIdIdentity.ts

```diff
@@ -5,6 +5,7 @@ export enum OpenIdIdentityIssuerType {
     AZUREAD = 'azuread',
     GENERIC_OIDC = 'oidc',
     SNOWFLAKE = 'snowflake',
+    SLACK = 'slack',
```

Comment:
Kind of confusing that we have this enum for `issuer_type` but we always query with `issuer` which is just a string and differs from `issuer_type`

![CleanShot 2025-07-04 at 17.54.45@2x.png](https://graphite-user-uploaded-assets-prod.s3.amazonaws.com/SRLEqMEevAzoFwvhnfeq/f0647454-9dc8-4a10-b26c-f2e6837a11a8.png)

Reply:
afaik we don't use the issuer for anything at the moment.

Reply:
okay no worries!